main.go

```diff
@@ -81,6 +84,9 @@ func printSplash() {
 }
 
 func main() {
+	if forceColor {
+		color.NoColor = false
+	}
```

Comment:
So here, you can use


```suggestion
	switch colorMode {
		case "always":
		color.NoColor = false
		case "never":
		color.NoColor = true
		case "auto", "":
		// do nothing
		default:
		log.Fatal("unsupported color mode: use always, never, auto")
	}
```